spec/lib/fixtures/state-jobless-properties.yml

```diff
@@ -35,3 +35,29 @@ pod:
     podIP: 192.0.2.2
     containerStatuses:
     - imageID: docker://bbb
+- metadata:
+    uid: 4123b33d-8e32-4d9c-baa5-c24704e830c3
+    name: new-pod-0
+    namespace: namespace
+    labels:
+      app.kubernetes.io/component: dummy
+    annotations: {}
+  spec:
+    containers:
+    - image: docker.io/image-one
+    - image: docker.io/image-two
+  status:
+    podIP: 192.0.2.2
+    containerStatuses:
+    - imageID: docker://bbb
+
+secret:
+- metadata:
+    name: new-pod-0-4123b33d-8e32-4d9c-baa5-c24704e830c3
+    namespace: namespace
+  data:
+    # echo '{"prop": "c"}' | base64
+    skiff-exported-properties-dummy: "eyJwcm9wIjogImMifQo="
+    # NOT the correct digest!
```

Comment:
That means there's a trailing new line here. Doesn't matter as far as I can tell, though.